src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanOrchestrator.hpp

```diff
@@ -27,15 +27,18 @@ constexpr auto INVENTORY_DB_PATH = "queue/vd/inventory";
 constexpr auto DELAYED_EVENTS_BULK_SIZE {1};
 constexpr auto DELAYED_QUEUE_PATH = "queue/vd/delayed";
 constexpr auto DELAYED_POSTPONE_SECONDS {60};
+constexpr auto MAX_THREADS {1};
```

Comment:
Compilation is breaking if I provide more than one thread

Its breaking in [this push](https://github.com/wazuh/wazuh/blob/98ee756f4b160b298734936aac708bfa1a755df3/src/shared_modules/utils/threadEventDispatcher.hpp#L315-L316), because it expects two parameters